app/src/main/res/values-ca/strings.xml

```diff
@@ -32,6 +33,7 @@
     <!--Install-->
     <string name="keep_force_encryption">Mantenir el xifrat forçat</string>
     <string name="keep_dm_verity">Mantenir AVB 2.0/dm-verity</string>
+    <string name="patch_vbmeta">Modifiqueu vbmeta en l'imatge d'inici</string>
```

Comment:
```suggestion
    <string name="patch_vbmeta">Modifiqueu vbmeta en l\'imatge d\'inici</string>
```
Apostrophes need to be escaped with `\`, that's probably why your builds are failing. Do the same change for other strings as well.